vaccine/tests/test_vaccine_reg_ussd.py

```diff
@@ -1105,9 +1107,10 @@ async def test_terms_and_conditions_3_invalid():
 
 @pytest.mark.asyncio
 async def test_state_success(evds_mock):
+    evds_mock.app.errormax = 1
```

Comment:
I'm not sure what this is for? I think we've got a separate test to test this

Reply:
Yeah, it was a copy/paste error. fixing